qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -489,6 +495,14 @@ def short_name(self) -> str:
     def t_complexity(self) -> 'TComplexity':
         return t_complexity(LessThanEqual(self.a_bitsize, self.b_bitsize))
 
+    def wire_symbol(self, soq: Soquet) -> WireSymbol:
+        if soq.name == 'a':
```

Comment:
soquet doesn't have a name field (?)

Reply:
yikes, should be soq.reg.name. I'll add a test.